app/services/platform/submitter_payload.rb

```diff
@@ -119,6 +119,20 @@ def csv_action
       }
     end
 
+    def confirmation_email_action
+      return if user_inputted_email.blank?
```

Comment:
```suggestion
      return if confirmation_email_answer.blank?
```

just thinking about making it specific to the confirmation email as opposed to any other email answer

Reply:
all done